hazelcast/protocol/codec/map_remove_all_codec.py

```diff
@@ -0,0 +1,17 @@
+from hazelcast.protocol.client_message import OutboundMessage, REQUEST_HEADER_SIZE, create_initial_buffer
+from hazelcast.protocol.builtin import StringCodec
+from hazelcast.protocol.builtin import DataCodec
+
+# hex: 0x013E00
+_REQUEST_MESSAGE_TYPE = 81408
+# hex: 0x013E01
+_RESPONSE_MESSAGE_TYPE = 81409
+
+_REQUEST_INITIAL_FRAME_SIZE = REQUEST_HEADER_SIZE
+
+
+def encode_request(name, predicate):
```

Comment:
Can you also create a PR in the protocol repo with the changes you did to generate this codec?

Reply:
Sure, I created👍